prime-react-demo/src/main/scala/react/table/demo/DemoComponents.scala

```diff
@@ -473,229 +488,224 @@ object DemoComponents {
                     )
                   )
                 )
-                // )
```

Comment:
Nothing below here was changed. Deleting the two commented out slashes just confused git.